frontend/src/app/person-dialog/person-dialog.component.ts

```diff
@@ -16,4 +16,8 @@ export class PersonDialogComponent {
     public dialogRef: MatDialogRef<PersonDialogComponent>,
     @Inject(MAT_DIALOG_DATA) public data: PersonDialogData,
   ) {}
+
+  isValid(data: PersonDialogData): boolean {
+    return data.firstName.length === 0 || data.lastName.length === 0;
+  }
```

Comment:
**issue:** the implementation is for a method called `isNotValid`, right? I'd suggest changing this method to

```suggestion
  isValid(data: PersonDialogData): boolean {
    return data.firstName.length > 0 && data.lastName.length > 0;
  }
```